src/test/resources/large/.gitattributes

```diff
@@ -31,3 +31,8 @@ dbsnp_138.b37.20.21.vcf.blockgz.gz.tbi filter=lfs diff=lfs merge=lfs -text
 tiledb_bin filter=lfs diff=lfs merge=lfs -text
 VQSR/** filter=lfs diff=lfs merge=lfs -text
 human_g1k_v37.20.21.fasta.img filter=lfs diff=lfs merge=lfs -text
+Homo_sapiens_assembly38.20.21.fasta filter=lfs diff=lfs merge=lfs -text
+gvcfs/HG00096.g.vcf.gz filter=lfs diff=lfs merge=lfs -text
+gvcfs/HG00268.g.vcf.gz filter=lfs diff=lfs merge=lfs -text
+gvcfs/NA19625.g.vcf.gz filter=lfs diff=lfs merge=lfs -text
+gvcfs/expected.combined.g.vcf.gz filter=lfs diff=lfs merge=lfs -text
```

Comment:
Can you include the origin in the name? (Eg., if it comes from GATK3 include the version of GATK3 that generated it in the name.